new-website/src/components/location/index.jsx

```diff
@@ -61,6 +61,21 @@ export const LocationOverview = () => {
         })
     }
 
+    const onSuccess = (msg) => {
+        setAlert({
+            open: true,
+            message: formatMessage({ id: msg }),
+            severity: "success",
+        })
+    }
+
+    React.useEffect(() => {
+        if (state?.saved == "success") {
+            state.saved = "reported" // Feels like a hack
```

Comment:
why do you need to do this hack?

Reply:
It avoids an update loop that was crashing execution eventually. IIUC it happened like this: Render Page → Effect → Trigger Alert (Re-render?) → Effect → Trigger Alert (Re-render?) → Effect → ...

Does it make sense? Likely to face a symptom of a different problem here, that can be avoided more easily.

Reply:
This sounds like a "dependency array" I might fix it:
>  Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.

Reply:
https://github.com/ImmunHelden/ImmunHelden.de/pull/155/files#r450441118

Reply:
Done. Used empty dependency array.